src/dotnet/APIView/APIView/Languages/CodeFileBuilder.cs

```diff
@@ -115,6 +118,31 @@ public CodeFile Build(IAssemblySymbol assemblySymbol, bool runAnalysis)
             return node;
         }
 
+        public static void BuildDependencies(CodeFileTokensBuilder builder, List<DependencyInfo> dependencies)
+        {
+            if (dependencies != null)
+            {
+                builder.NewLine();
+                builder.Append("Dependencies:", CodeFileTokenKind.Text);
+                builder.NewLine();
+                foreach (DependencyInfo dependency in dependencies)
+                {
+                    builder.Append(new CodeFileToken(dependency.Name, CodeFileTokenKind.Text)
+                    {
+                        // allow dependency to be commentable
+                        DefinitionId = dependency.Name
+                    });
+                    // don't include the version in the API sign-off diffs
+                    builder.Append(null, CodeFileTokenKind.SkipDiffRangeStart);
+                    builder.Append($"-{dependency.Version}", CodeFileTokenKind.Text);
```

Comment:
So, this will require API review approval from architect even if minor version is changed for dependency. Do you want APIView to ignore this part from comparing when it compares nightly artifact with existing automatic API revision or is it supposed to create a new revision and force API review to get reviewed? If preference is to skip comparison and avoid new revision then version part should be surrounded by skip diff tags.

Reply:
Yes, I thought that is what I was doing here 😄 

The version token is surrounded by SkipDiff tags.

Reply:
Oh I think I am becoming blind. I didn't see it earlier. It looks fine.